ksql-parser/src/test/java/io/confluent/ksql/parser/tree/ExpressionTreeRewriterUnitTest.java

```diff
@@ -0,0 +1,539 @@
+package io.confluent.ksql.parser.tree;
```

Comment:
I am planning on replacing ExpressionTreeRewriterTest with this test. Wanted to get a green run with the old tests first.

Reply:
Should we do this before merging?